pkg/apis/machine/types.go

```diff
@@ -271,6 +286,29 @@ const (
 	ConditionUnknown ConditionStatus = "Unknown"
 )
 
+const (
+	// NodeInPlaceUpdate is a node condition type for in-place update
+	NodeInPlaceUpdate corev1.NodeConditionType = "InPlaceUpdate"
+
+	// UpdateCandidate is a constant for reason in condition that indicates node is candidate for update
+	UpdateCandidate string = "UpdateCandidate"
```

Comment:
to follow the naming convention that you have for `SelectedForUpdate`, `ReadyForUpdate` - should this be renamed to `CandidateForUpdate`?

Reply:
Addressed in 9f57ff6a75287a81d0354aafaf24270eb7e2df88